src/Middleware/Diagnostics/src/DeveloperExceptionPage/IDeveloperPageExceptionFilter.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.Diagnostics
```

Comment:
Add a sample that implements the interface, does content negotiation, and returns a JSON formatted error.

Reply:
Does this repo have samples that aren't tests? I haven't found them. Where are they?

Reply:
Oh, I didn't realize those had been dropped. We used to have all of these: https://github.com/aspnet/Diagnostics/tree/master/samples

Reply:
Good old days. It is unfortunate we have no way to f5 this.